src/cli/Edit.cpp

```diff
@@ -143,14 +172,39 @@ int Edit::execute(const QStringList& arguments)
     } else if (parser.isSet(generate)) {
         PasswordGenerator passwordGenerator;
 
+        PasswordGenerator::CharClasses classes = 0x0;
+
+        classes |= PasswordGenerator::DefaultCharset;
+
+        if (parser.isSet(numeric)) {
```

Comment:
@AnatomicJC Isn't `numeric` already in the default charset?? see https://github.com/keepassxreboot/keepassxc/blob/develop/src/core/PasswordGenerator.h#L44

Also, shouldn't we remove completely the default charset if character classes are explicit in the options? That would imply adding options for `LowerLetters` and `UpperLetters` too.

Reply:
Hi @louib sorry for the delay, I will try to do this before end of this week.

Reply:
Hi @louib I removed numeric option as it is already part of the default charset. The "l" parameter for add/edit is already assigned to the password length. To add `LowerLetters`, I should add another "L" param. It can be confusing, no ? What do you think ?

Reply:
@AnatomicJC I had a look at what the `Generate` command does, and it has `-L` for the password length and `-l` for lower case letters. Whatever we decide between those 2 options it should be the same in `Add`, `Edit` and `Generate`. This means we will introduce a breaking change for either `Generate` or `Add` and `Edit`, but this PR is going into 2.5.0 so we can live with that.